engines/wintermute/ad/ad_scene.cpp

```diff
@@ -568,6 +589,9 @@ TOKEN_DEF(LAYER)
 TOKEN_DEF(WAYPOINTS)
 TOKEN_DEF(EVENTS)
 TOKEN_DEF(CURSOR)
+#ifdef ENABLE_WME3D
```

Comment:
It probably doesn't hurt to enable all the tokens even in the 2D-case, and instead commenting about which ones are 3D-only and not.